docs/source/gateway.md

```diff
@@ -0,0 +1,118 @@
+# Gateway
+
+## Overview
+
+The Gateway is a new service introduced in v2.4 that provides a simplified, minimal API for submitting transactions
+to a Fabric network.
+
+## Writing client applications
+
+Starting with v2.4, client applications should use one of the v2.4 SDKs which are optimised to interact with the Gateway.
+These SDKs expose the same high-level programming model which was first introduced with v1.4 of Fabric.
+
+The gateway supports the following actions
+- **Evaluate** (query) a transaction.  This will invoke a smart contract (chaincode) function on a single peer and return the result
+to the client.  The gateway will preferably select a peer in the same organization as the gateway peer and choose the peer 
+with the highest ledger block height.  If no peer is available in the gateway's organization, then it will choose a peer
+from another organization.
+- **Endorse** a transaction. This will gather enough endorsements to satisfy the combined signature policies (see below) and
+return a prepared transaction envelope to the client for signing.
```

Comment:
I'm thinking we should include the word "response" here since we talk about proposal response in other parts of the doc. However, I'm forgetting exactly how EndorserResponse, ProposalResponse, and PreparedTransaction are related. Could you remind me :-)

Reply:
Changed to `This will gather enough endorsement responses to satisfy...`
The various response structures you mention are just the protobuf messages that get returned from the various gateway grpc methods.  They are wrappers for the peer ProposalResponse, etc messages and contain some other information. Details here https://github.com/hyperledger/fabric-protos/blob/main/gateway/gateway.proto
I wasn't intending to document the proto level API in this docs topic.